adapter/ryu/omniui/omniui.py

```diff
@@ -97,10 +573,71 @@ def getFlows(self, dpid):
         else:
             LOG.debug('Unsupported OF protocol')
             return None
+
+        print '----------------------flowAPI----------------------'
+
+        flowstatsReplyAPI = {}
+        flowstatsReplyAPI["controller"] = controllerName
+
+        for key in flows:
+            temp = "%016x" %int(key)
+            temp = map(str, temp)
+            for i in range(2, 23, 3):
+                temp.insert(i, ':')
+            flowstatsReplyAPI["dpid"] = ''.join(temp)
+
+            flowstatsReplyAPI["flows"] = []
+            i = 0
+            for inflow in flows[key]:
+                if inflow["priority"] == 1:
```

Comment:
In my testing environment, the priority of flows are 65535. So all the flow processing things below this "if" condition are skipped. Therefore this handler posted an empty flows dictionary to Core.